containers/go-postgres/.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,43 @@
+{	
+	"name": "Go & PostgreSQL",
+	"dockerComposeFile": "docker-compose.yml",
+	"service": "app",
+	"workspaceFolder": "/workspace",
+
+	// Set *default* container specific settings.json values on container create.
```

Comment:
This is basically copied in from the other files and containers. I do agree with you, but since the json is only being read by VSCode, I think the idea is that the documentation is worth having it fail strict parsing.